tests/e2e/forms/repeat-form.wdio-spec.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+
+const constants = require('../../constants');
+const utils = require('../../utils');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+
+const xml = fs.readFileSync(`${__dirname}/../../../demo-forms/repeat-translation.xml`, 'utf8');
+const formDocument = {
+  _id: 'form:repeat-translation',
+  internalId: 'repeat-translation',
+  title: 'Repeat',
+  type: 'form',
+  _attachments: {
+    xml: {
+      content_type: 'application/octet-stream',
+      data: Buffer.from(xml).toString('base64')
+    }
+  }
+};
+const userContactDoc = {
+  _id: constants.USER_CONTACT_ID,
+  name: 'Jack',
+  date_of_birth: '',
+  phone: '+64274444444',
+  alternate_phone: '',
+  notes: '',
+  type: 'person',
+  reported_date: 1478469976421,
+  parent: {
+    _id: 'some_parent'
+  }
+};
+
+describe('RepeatForm', () => {
+  before(async () => {
+    await utils.seedTestData(userContactDoc, [formDocument]);
+  });
+
+  afterEach(async () => {
+    await browser.deleteCookies();
+    await browser.refresh();
+  });
+
+  it('should display the initial form and its repeated content in Swahili', async () => {
+    const swUserName = 'Jina la mtumizi';
+    await loginPage.changeLanguage('sw', swUserName);
+    await loginPage.cookieLogin();
+    await commonPage.goToReports();
+    await (await reportsPage.submitReportButton()).click();
+    await (await reportsPage.formLinkByHref(formDocument.internalId)).click();
+
+    const stateLabel = await $('#report-form .question-label.active[data-itext-id="/repeat/basic/state_1:label"]');
+    expect(await stateLabel.getText()).toBe('Select a state: - SV');
+
+    const addRepeatButton = await $('.btn.btn-default.add-repeat-btn');
+    await addRepeatButton.click();
+
+    const cityLabel = await $('#report-form .question-label.active[data-itext-id="/repeat/basic/rep/city_1:label"]');
+    expect(await cityLabel.getText()).toBe('Select a city: - SV');
+  });
+
+  it('should display the initial form and its repeated content in English', async () => {
+    const enUserName = 'User name';
+    await loginPage.changeLanguage('en', enUserName);
+    await loginPage.cookieLogin();
```

Comment:
Isn't it simpler to just pass the language parameter to `cookieLogin` (you've already added it as an additional parameter and then didn't actually use it) than to change the language in the UI and alter `cookieLogin` to account for it. 
As I see it now, `cookieLogin` is completely stateless, and doesn't even require the browser to be on the login page, and I think it should remain that way. 
We could possibly change the declaration so it's easier to pass the language parameters, for example have to function accept an "options" object instead of a list of params, so properties can be easier omitted.

Reply:
Since it's an E2E test, I figured it would make sense to flow through the same actions a real user would.  
I agree with you, `cookieLogin` should remain stateless. I'm making some changes right now

Reply:
There's an actual login procedure in the login page that logs in using the "button", instead of setting the cookie, you could use that instead to replicate user experience.

Reply:
Ah yes, that's much better, thanks!